Dockerfile_build_ubuntu64_go1.11

```diff
@@ -24,7 +24,10 @@ RUN gem install fpm
 
 # Install go
 ENV GOPATH /root/go
-ENV GO_VERSION 1.9.2
+
+# TODO(edd) this needs to be updated to 1.11 when the branch is available.
+ENV GO_VERSION 1.10.3
```

Comment:
Possibly we could use 1.11beta1 for now on this: https://golang.org/dl/#unstable

But I can live with it set to 1.10 for now.

Reply:
Yeah I don't fancy the grief. Let's wait for the 1.11 release, update to that, and then we can do testing on it until we ship Influx 1.7/1.8  with Go 1.11 if there have been no issues over the summer/autumn.